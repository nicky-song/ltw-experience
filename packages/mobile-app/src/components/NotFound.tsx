// TODO: move this to it's own component
import { Text, View } from 'react-native';
import Icon from 'react-native-vector-icons/AntDesign';

export const NotFound = () => (
  <View
    style={{
      justifyContent: 'center',
      alignItems: 'center',
      paddingTop: 100,
    }}>
    <Icon name={'frowno'} size={100} color={'#c0c0c0'} />
    <Text
      style={{
        marginTop: 20,
        fontSize: 20,
        fontWeight: 'bold',
        color: '#c0c0c0',
      }}>
      Page not found
    </Text>
  </View>
);
