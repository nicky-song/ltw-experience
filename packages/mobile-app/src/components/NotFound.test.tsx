import renderer from 'react-test-renderer';
import { NotFound } from './NotFound';

describe('NotFound Component', () => {
  it('matches snapshot', () => {
    const activityIndicator1 = renderer.create(<NotFound />);
    expect(activityIndicator1).toMatchSnapshot();
  });
});
