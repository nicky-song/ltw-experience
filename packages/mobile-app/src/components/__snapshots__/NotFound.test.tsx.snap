// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component matches snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "justifyContent": "center",
      "paddingTop": 100,
    }
  }
>
  <Text />
  <Text
    style={
      {
        "color": "#c0c0c0",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginTop": 20,
      }
    }
  >
    Page not found
  </Text>
</View>
`;
